src/components/layout/InformationBlocksContainer/InformationBlocksContainer.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import InformationBlocksContainer from './InformationBlocksContainer';
+import InformationBlock from '../../blocks/InformationBlock';
+
+export default {
+  title: 'Layout/InformationBlocksContainer',
+  component: InformationBlocksContainer,
+  argTypes: {},
+} as ComponentMeta<typeof InformationBlocksContainer>;
+
+const Template: ComponentStory<typeof InformationBlocksContainer> = (args) => {
+  const data = [
+    {
+      id: 123,
+      value: 'hello',
+      label: 'world',
+    },
+  ];
+
+  return (
+    <InformationBlocksContainer {...args}>
+      <InformationBlock
```

Comment:
Let's duplicate this block below so we can see how this container works with two/three blocks inside